lib/vagrant-ovirt4/action/read_ssh_info.rb

```diff
@@ -37,6 +37,10 @@ def read_ssh_info(env)
           nics_service = server.nics_service
           nics = nics_service.list
           ip_addr = nics.collect { |nic_attachment| env[:connection].follow_link(nic_attachment.reported_devices).collect { |dev| dev.ips.collect { |ip| ip.address if ip.version == 'v4' } unless dev.ips.nil? } }.flatten.reject { |ip| ip.nil? }.first rescue nil
+          if ip_addr.nil?
```

Comment:
it would be better here to use a begin rescue on line 39 then fall to this

perhaps remove the rescue nil from 39 and use this behavior? i can refactor after that change if you need

Reply:
In follow up commit, 4.4 handling moved to 'begin' and  4.3 to 'rescue'. That's how you meant it?

Reply:
If this works as expected for you, then yes

@daFritz84 can you confirm this continues to work with your recent fix?

Reply:
Yes, the fallback via rescue works with my 4.3 cluster.

Reply:
If @dafritz84 with 4.4 can confirm this works Ill merge it